content/productivity/crawl-planets/plugin.js

```diff
@@ -3,233 +3,328 @@
+        stepper.min = '0';
+        stepper.max = '100';
+        stepper.step = '5';
+        stepper.value = `${this.maxEnergyPercent}`;
+        stepper.style.width = '80%';
+        stepper.style.height = '24px';
+
+        let percent = document.createElement('span');
+        percent.innerText = `${stepper.value}%`;
+        percent.style.float = 'right';
+
+        stepper.onchange = (evt) => {
+            percent.innerText = `${evt.target.value}%`;
+            try {
+                this.maxEnergyPercent = parseInt(evt.target.value, 10);
+            } catch (e) {
+                console.error('could not parse energy percent', e);
+            }
+        }
+
+        let minimumEnergyAllowedLabel = document.createElement('label');
+        minimumEnergyAllowedLabel.innerText = 'Allow minimum energy to capture?';
+        minimumEnergyAllowedLabel.style.display = 'block';
+
+        let minimumEnergyAllowedSelect = document.createElement('select');
+        minimumEnergyAllowedSelect.style.background = 'rgb(8,8,8)';
+        minimumEnergyAllowedSelect.style.width = '100%';
+        minimumEnergyAllowedSelect.style.marginTop = '10px';
+        minimumEnergyAllowedSelect.style.marginBottom = '10px';
+        states.forEach(state => {
+            let opt = document.createElement('option');
+            opt.value = `${state}`;
+            opt.innerText = `${state}`;
+            minimumEnergyAllowedSelect.appendChild(opt);
+        });
+        minimumEnergyAllowedSelect.value = `${this.minimumEnergyAllowed}`;
+
+        minimumEnergyAllowedSelect.onchange = (evt) => {
+            try {
+                let bool_value = evt.target.value == "true" ? true : false
+                this.minimumEnergyAllowed = bool_value;
+            } catch (e) {
+                console.error('could not parse minimumEnergyAllowed boolean', e);
+            }
+        }
+
+        let levelLabelTo = document.createElement('label');
+        levelLabelTo.innerText = 'Min. level to capture';
+        levelLabelTo.style.display = 'block';
+
+        let levelTo = document.createElement('select');
+        levelTo.style.background = 'rgb(8,8,8)';
+        levelTo.style.width = '100%';
+        levelTo.style.marginTop = '10px';
+        levelTo.style.marginBottom = '10px';
+        planetLevels.forEach(lvl => {
+            let opt = document.createElement('option');
+            opt.value = `${lvl}`;
+            opt.innerText = `Level ${lvl}`;
+            levelTo.appendChild(opt);
+        });
+        levelTo.value = `${this.minPlanetLevelTo}`;
+
+        levelTo.onchange = (evt) => {
+            try {
+                this.minPlanetLevelTo = parseInt(evt.target.value, 10);
+            } catch (e) {
+                console.error('could not parse planet level', e);
+            }
+        }
+
+        let levelLabelFromMin = document.createElement('label');
+        levelLabelFromMin.innerText = 'Min. level to send energy from';
```

Comment:
If were going to add these, Id put them UNDER the crawl from selected button since theyre not related to crawl from selected. That might make it more clear

Reply:
Correct, resolved it